CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-
+# dbt_ad_reporting v1.7.0
+[PR #103](https://github.com/fivetran/dbt_ad_reporting/pull/103) includes the following updates.
+## Feature update 🎉
+- Unioning capability! This adds the ability to union source data from multiple ad reporting connectors. Refer to the [README](https://github.com/fivetran/dbt_ad_reporting/blob/main/README.md) for more details.
```

Comment:
Actually, can we remove the README "hide" sections in these updates. This way we are able to link directly to the section in the README where the users needs to understand how to configure the union feature.

Otherwise, I worry this cool feature is hidden from people actually using it.